### DISCUSSION TITLE
How to name the new child placement property and options

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/freeplane#712] [General]
Since v1.10.5-pre01, there is a new property in **Tool panel->Format**

![2022-10-02_09,45,55](https://user-images.githubusercontent.com/7197131/193445886-2b8c8dca-f729-40ea-a8f1-4286789e6417.png)

It's meant to be compatible with the current node layout (left-right) and a future up-down layout.

![2022-10-02_09,32,44](https://user-images.githubusercontent.com/7197131/193446050-2a50bccd-4470-43b6-a102-766baa3e9746.png)

![2022-10-02_10,53,52](https://user-images.githubusercontent.com/7197131/193446253-192339dd-de4e-49d9-ae11-de45a5f2e9ef.png)

The initially proposed options are on the left/top, i.e.

**Child placement:**
- by the first node
- center
- by the last node

I think the ones on the right/down are more fitting, i.e. 

**Child alignment:**
- align first child with parent
- center
- align last child with parent

What do you think? Any other ideas?

**Comment by ldmpub** [comment]
I agree with your wording proposals and have already implemented it for french, for instance: "Align first child and parent"

**Comment by dpolivaev** [comment]
I changed it to get the meaningful word at the beginning which is important when the panel is narrow.

<img width="111" alt="alignment" src="https://user-images.githubusercontent.com/1558052/193450170-1fe740c1-9e9c-4271-b3b6-88823d3739a0.png">
